fs/zipfs.py

```diff
@@ -456,3 +468,16 @@ def geturl(self, path, purpose="download"):
             return "zip://{}!/{}".format(quoted_file, quoted_path)
         else:
             raise errors.NoURL(path, purpose)
+
+    def setpassword(self, password):
+        # type: (bytes) -> None
+        """Set *password* as default password to extract encrypted files.
+        """
+        _password_type_check(password)
+        self._zip.setpassword(password)
+
+
+def _password_type_check(password):
+    if isinstance(password, six.binary_type):
+        return
+    raise TypeError("except bytes for password, not " + type(password).__name__)
```

Comment:
Grammar nit: "Accept" not "Except"

Reply:
Or maybe it was a typo of "expect bytes..." ? :man_shrugging:

Reply:
yeah it's typo 😆 i'd like to type 'expect'

Reply:
> Or maybe it was a typo of "expect bytes..." ?

That never occurred to me hahaha

Reply:
So it should ultimately say 'Expected', I guess.

Reply:
That would make more sense than my suggestion.